Marlin/src/feature/leds/neopixel.h

```diff
@@ -110,9 +110,7 @@ class Marlin_NeoPixel {
       #if CONJOINED_NEOPIXEL
         adaneo2.show();
       #else
-      #if DISABLED(NEOPIXEL2_SEPARATE)
-        adaneo1.setPin(NEOPIXEL2_PIN);
-      #endif
+        TERN_(NEOPIXEL2_SEPARATE,,adaneo1.setPin(NEOPIXEL2_PIN));
```

Comment:
`TERN_(NEOPIXEL2_SEPARATE,,adaneo1.setPin(NEOPIXEL2_PIN));` has an extra comma in it
and dosent work as is